### DISCUSSION TITLE
problem with building android app using quasar in cordova mode

### DISCUSSION THREAD

**Discussion by enurt** [quasarframework/quasar#11935] [CLI - Capacitor / Cordova modes]
hello, after running the 'quasar dev -m cordova -T android' command i got this error "Unable to get mutable Windows environment variable map" which i cannot solve. i think that i already set all path correctly. please help

**Comment by wwwizzarrdry** [comment]
This fixed the error, "Unable to get mutable Windows environment variable map" for me.
![environment1](https://user-images.githubusercontent.com/2218144/167265524-6cbc22dd-34af-4bad-ad62-63919f9e9f44.png)

**Reply by JayFett** [reply]
For anyone else that runs into this problem now or in the future...  the solution for me was to install Java 8 rather than 18.
